docs/content/preview/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/kubernetes.md

```diff
@@ -181,49 +185,79 @@ You can create a `kubeconfig` file for the previously created `yugabyte-platform
-5. Use **Pull Secret File** to upload the pull secret to download the image of the Enterprise YugabyteDB that is in a private repository. Your Yugabyte sales representative should have provided this secret.
+Use **Kube Config** to upload the `kubeconfig` file. If specified, this configuration file is used for all availability zones in all regions.
 
-## Configure region and zones
+Alternately, you can define separate `kubeconfig` files for each zone when defining the regions. See [Configure region and zones](#configure-region-and-zones).
 
-Continue configuring your Kubernetes provider by clicking **Add region** and completing the **Add new region** dialog shown in the following illustration: 
+### Configure region and zones
 
-![Add new region](/images/ee/k8s-setup/k8s-az-kubeconfig.png)
+Continue configuring your Kubernetes provider by clicking **Add region** and completing the **Add new region** dialog as follows:
 
 1. Use the **Region** field to select the region.
 
-1. Use the **Zone** field to select a zone label that should match the value of failure domain zone label on the nodes. `topology.kubernetes.io/zone` would place the pods in that zone.
+1. Click **Add Zone** and complete the corresponding portion of the dialog. Notice that there are might be multiple zones.
+
+1. Use the **Zone** field to select a zone label that should match the value of failure domain zone label on the nodes. For example, `topology.kubernetes.io/zone` would place the pods in that zone.
+
+1. Use **Kube Config** to upload the `kubeconfig` file. If this file is available at the provider level, you are not required to supply it.
 
-1. Optionally, use the **Storage Class** field to enter a comma-delimited value. If you do not specify this value, it would default to standard. You need to ensure that this storage class exists in your Kubernetes cluster and takes into account [storage class considerations](../../../install-yugabyte-platform/prepare-environment/kubernetes/#configure-storage-class).
+1. Optionally, use the **Storage Classes** field to enter a comma-delimited value. If you do not specify this value, it would default to standard. You need to ensure that this storage class exists in your Kubernetes cluster and takes into account [storage class considerations](../../../install-yugabyte-platform/prepare-environment/kubernetes/#configure-storage-class).
 
-1. Use the **Namespace** field to specify the namespace. If provided service account has the `Cluster Admin` permissions, you are not required to complete this field. The service account used in the provided `kubeconfig` file should have access to this namespace.
+1. In the **Kube Domain** field, provide the DNS domain name used in the Kubernetes cluster.
```

Comment:
There is "Kube Pod Address Template (Optional)" after Storage Classes. You need to add this field desc. Checked in portal.dev..

Reply:
Also , Kube Domain, Kube Namespace, and Overrides are optional fields as well.